### DISCUSSION TITLE
Styled map data in a flutter application

### DISCUSSION THREAD

**Discussion by kvo87** [organicmaps/organicmaps#5147] [Q&A]
Hello, I'm working on my own flutter application for travelers which should combine planning with POI saving, expense logger and other useful features that may help in a trip. Previously I actively used maps.me as a traveler, then switched to organic maps and likes it very much.  So I'm curious, is it possible to get map data with same styles as in Organic maps to display them in flutter OSM viewer?

**Comment by biodranik** [comment]
Everything is possible, but it may require a lot of your efforts to build and use the code.

**Comment by h3x4d3c1m4l** [comment]
Take a look at the vector map plugins for https://pub.dev/packages/flutter_map. This way you can load map data but should be able to style your own way.
